framework/src/utils/PetscDMMoose.C

```diff
@@ -1496,7 +1508,7 @@ DMView_Moose(DM dm, PetscViewer viewer)
     CHKERRQ(ierr);
     for (const auto & bit : *(dmm->_block_ids))
     {
-      ierr = PetscViewerASCIIPrintf(viewer, "(%s,%D) ", bit.first.c_str(), bit.second);
+      ierr = PetscViewerASCIIPrintf(viewer, "(%s,%d) ", bit.first.c_str(), bit.second);
```

Comment:
I believe this should be unsigned. I only thought of this because you changed to `u` for the `_var_ids`

Reply:
Oops never mind I didn't read `id_types.h` close enough. Nothing to see here